packages/editor/src/components/post-actions/actions.js

```diff
@@ -363,10 +300,6 @@ export function usePostActions( { postType, onActionPerformed, context } ) {
 				  ! isPattern &&
 				  duplicatePostAction
 				: false,
-			isTemplateOrTemplatePart &&
-				userCanCreatePostType &&
-				isBlockBasedTheme &&
```

Comment:
@t-hamano Just noting that this was missing from the dependencies of this hook (it's not needed now though)